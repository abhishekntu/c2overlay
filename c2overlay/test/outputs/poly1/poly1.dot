digraph graphname {
N8 [color="black", label="N7N5_sub_N7_mul_N5_N8"];
N9 [color="black", label="O0_N12_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="N11_sub_Imm_1_N11_N3"];
N4 [color="black", label="N8_sqr_N8_N4"];
N5 [color="black", label="N9_sqr_N9_N5"];
N6 [color="black", label="N4_add_Imm_1_N4_N6"];
N7 [color="black", label="N6N3N10_add_N6_mul_N3_sub_N10_N7"];
N8 -> N7;
N1 -> N5;
N1 -> N6;
N1 -> N3;
N2 -> N8;
N2 -> N7;
N2 -> N4;
N3 -> N7;
N4 -> N8;
N5 -> N7;
N6 -> N8;
N7 -> N9;
}
