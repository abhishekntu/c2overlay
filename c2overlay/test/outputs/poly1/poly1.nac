procedure foo(in s32 x,in s32 y,out s32 D_2446)
{
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= add x,1;
  t1_2 <= sqr y;
  t2_1 <= sqr x;
  t2_2 <= sub x,1;
  t2_3 <= sub t1_1,y;
  t3_1 <= add t2_2,y;
  t3_2 <= mul t2_3,t1_2;
  t4_1 <= mul t2_1,t3_1;
  D_2446 <= sub t4_1,t3_2;
}
