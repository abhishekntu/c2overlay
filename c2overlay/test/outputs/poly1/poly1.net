.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.output out:N9
pinlist: N9
    
.clb N8_blk
pinlist: N3 N5 N2 open N8 open open open open 
subblock: N8_blk 0 1 2 open 4 open open open open
    
.clb N3_blk
pinlist: N1 open open open N3 open open open open 
subblock: N3_blk 0 open open open 4 open open open open

.clb N4_blk
pinlist: N2 open open open N4 open open open open 
subblock: N4_blk 0 open open open 4 open open open open

.clb N5_blk
pinlist: N1 open open open N5 open open open open 
subblock: N5_blk 0 open open open 4 open open open open

.clb N6_blk
pinlist: N1 N2 open open N6 open open open open 
subblock: N6_blk 0 1 open open 4 open open open open

.clb N7_blk
pinlist: N4 N6 N8 open N9 open open open open 
subblock: N7_blk 0 1 2 open 4 open open open open
    
