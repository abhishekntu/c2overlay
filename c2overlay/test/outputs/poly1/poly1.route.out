Array size: 5 x 5 logic blocks.

Routing:

Net 0 (N1)

SOURCE (6,1)  Pad: 1  
  OPIN (6,1)  Pad: 1  
 CHANY (5,1)  Track: 1  
  IPIN (5,1)  Pin: 1  
  SINK (5,1)  Class: 0  
 CHANY (5,1)  Track: 1  
 CHANX (5,0)  Track: 1  
 CHANX (4,0)  Track: 1  
  IPIN (4,1)  Pin: 2  
  SINK (4,1)  Class: 0  
 CHANX (4,0)  Track: 1  
 CHANY (3,1)  Track: 0  
 CHANY (3,2)  Track: 0  
  IPIN (4,2)  Pin: 3  
  SINK (4,2)  Class: 0  


Net 1 (N2)

SOURCE (2,0)  Pad: 1  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 1  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 0  
 CHANX (3,0)  Track: 0  
  IPIN (3,1)  Pin: 2  
  SINK (3,1)  Class: 0  
 CHANX (2,0)  Track: 1  
 CHANX (1,0)  Track: 1  
  IPIN (1,1)  Pin: 2  
  SINK (1,1)  Class: 0  


Net 2 (N8)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 5  
 CHANY (2,1)  Track: 1  
  IPIN (3,1)  Pin: 3  
  SINK (3,1)  Class: 0  


Net 3 (N3)

SOURCE (4,2)  Class: 1  
  OPIN (4,2)  Pin: 7  
 CHANY (3,2)  Track: 1  
 CHANY (3,1)  Track: 1  
  IPIN (3,1)  Pin: 1  
  SINK (3,1)  Class: 0  


Net 4 (N4)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 5  
 CHANY (1,1)  Track: 1  
  IPIN (2,1)  Pin: 3  
  SINK (2,1)  Class: 0  


Net 5 (N5)

SOURCE (4,1)  Class: 1  
  OPIN (4,1)  Pin: 4  
 CHANX (4,1)  Track: 1  
 CHANX (3,1)  Track: 1  
  IPIN (3,1)  Pin: 0  
  SINK (3,1)  Class: 0  


Net 6 (N6)

SOURCE (5,1)  Class: 1  
  OPIN (5,1)  Pin: 4  
 CHANX (5,1)  Track: 1  
 CHANY (4,2)  Track: 0  
 CHANX (4,2)  Track: 1  
 CHANX (3,2)  Track: 1  
 CHANY (2,2)  Track: 1  
 CHANX (2,1)  Track: 1  
  IPIN (2,1)  Pin: 0  
  SINK (2,1)  Class: 0  


Net 7 (N9)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 6  
 CHANX (3,0)  Track: 1  
  IPIN (3,0)  Pad: 0  
  SINK (3,0)  Pad: 0  
